moteur-server-rest/routes.py

```diff
@@ -80,3 +81,9 @@ def handle_status(workflow_id):
             workflow_status = "UNKNOWN"
     print(workflow_status)
     return workflow_status
+
+
+@app.route("/")
+def index():
+    app.logger.debug("Call to /")
+    return "Hello, World!"
```

Comment:
print a better message, like `this is moteut_server_rest`. Maybe add the version if possible.

Reply:
ok, we'll deal with that later and add a github issue.